packages/components/resource-pages/ResourcePopOver.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
+import ResourceLink from '../../utils/generics/resource-link';
+
+const ResourcePopOver: React.FC<ResourcePopOverProps> = ({
+  position,
+  children,
+  label,
+  title,
+}) => {
+  return (
+    <Popover
+      position={PopoverPosition.auto || position}
+      headerContent={title}
+      bodyContent={children}
+      aria-label={title}
+    >
+      <Button variant="link" isInline>
+        {label}
+      </Button>
+    </Popover>
+  );
+};
+
+export const DataResourcesPopOver: React.FC<DataResourcesPopOverProps> = ({
+  label,
+  dataResources,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <ResourcePopOver label={label} title={t('Connected data sources')}>
+      <div className="resource-pop-over">
+        {dataResources?.map((resourceName) => (
+          <ResourceLink
+            link={`/mcgms/resource/noobaa.io~v1alpha1~NamespaceStore/${resourceName}`}
```

Comment:
Replace: ```noobaa.io~v1alpha1~NamespaceStore``` by util function like getGVK and referenceFor

Reply:
is NamespaceStore is not namespace scope or cluster scope? i dont see any namespace in link

Reply:
it is namespace scoped... but we only create it in the NS where noobaa is deployed... so it will be always same...
that is why we don't have it in the link as well (same as we do in ODF)...